docs/pbc/all/search/algolia.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Algolia
+description: Algolia is a search engine that 
+---
+
+By default, all Spryker stores have [Elasticsearch](https://www.elastic.co/elasticsearch/) as their search engine. However, you can replace it with the Algolia one.
+
+The Algolia search engine that stands out from others due to its performance. With the Algolia app, your users can conduct advanced search of active concrete products in your store. 
+
+{% info_block infoBox "Product offers and product reviews" %}
```

Comment:
This section is not true, in my opinion. We only send concrete product information; no product offers are sent there.
@matweew can you confirm?